pymatgen/transformations/standard_transformations.py

```diff
@@ -680,15 +680,20 @@ class PerturbStructureTransformation(AbstractTransformation):
     Args:
         amplitude (float): Amplitude of perturbation in angstroms. All sites
             will be perturbed by exactly that amplitude in a random direction.
+        min_random_distance (None, int, or float): If None, perturb
```

Comment:
For the transformation, can you update this to be `min_random_amplitude` to match the other argument names.

Also edit the rest of the docstring accordingly.